tests/integration/aps/test_aps_api_client.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+
+import pytest
+from aps.aps_api_client import APSApiClient
+from aps.aps_params import APSParams
+
+
+@pytest.fixture(scope="session")
+def vcr_config():
+    return {
+        "ignore_localhost": True,
+        "decode_compressed_response": True,
+        "filter_headers": ("X-Amz-Date"),
+        "record_mode": "once",
+    }
+
+
+@pytest.fixture(scope="function")
+def aps_api_client_fixture():
+    yield APSApiClient()
+
+
+@pytest.mark.vcr(record_mode="once")
+def test_get_articles_metadata(aps_api_client_fixture, shared_datadir):
+    parameters = APSParams(
+        from_date="2021-04-10", until_date="2022-04-10", per_page=1
+    ).get_params()
+    metadata_value = aps_api_client_fixture.get_articles_metadata(parameters=parameters)
+    file_path = os.path.join(shared_datadir, "test_json.json")
+    with open(file_path, "rb") as json_file:
+        content = json_file.read()
+        assert json.loads(content) == metadata_value
+
+
+@pytest.mark.vcr(record_mode="once")
+def test_get_pdf_file(aps_api_client_fixture, shared_datadir):
+    doi = "10.1103/PhysRevLett.126.153601"
+    parameters = APSParams(from_date="2021-04-10", until_date="2022-04-10").get_params()
+    pdf = aps_api_client_fixture.get_pdf_file(parameters=parameters, doi=doi)
+    file_path = os.path.join(shared_datadir, "test_pdf.pdf")
+    with open(file_path, "rb") as pdf_file:
+        assert pdf_file.read() == pdf
+
+
+@pytest.mark.vcr(record_mode="once")
```

Comment:
since you have a global config for `vcrpy` you don't need to specify the `once` in every test, could you please remove it?